src/systems/systems/src/hydraulic/landing_gear.rs

```diff
@@ -372,17 +384,18 @@ impl GearSystemComponentAssembly {
     const UPLOCKED_PROXIMITY_DETECTOR_TRIG_DISTANCE_RATIO: f64 = 0.01;
 
     fn new(
-        component_id: GearSysComponentId,
+        id: GearActuatorId,
         is_inverted_control: bool,
         hydraulic_assembly: HydraulicLinearActuatorAssembly<1>,
         has_hydraulic_downlock: bool,
         uplock_id: [ProximityDetectorId; 2],
         downlock_id: [ProximityDetectorId; 2],
     ) -> Self {
         let mut obj = Self {
-            component_id,
+            component_id: GearSysComponentId::from(id),
```

Comment:
Or simply:
```suggestion
            component_id: id.into(),
```

Reply:
right always forget about this one available when implementing from !